src/java/htsjdk/samtools/Defaults.java

```diff
@@ -73,6 +80,7 @@
             NON_ZERO_BUFFER_SIZE = BUFFER_SIZE;
         }
         REFERENCE_FASTA = getFileProperty("reference_fasta", null);
+        USE_CRAM_REF_DOWNLOAD = getBooleanProperty("ref_download_enabled", false);
```

Comment:
Can the property name that you pass to `getBooleanProperty()` match the variable name? Eg.,

    USE_CRAM_REF_DOWNLOAD = getBooleanProperty("use_cram_ref_download", false);